src/mocks/filters.ts

```diff
@@ -0,0 +1,79 @@
+import { isPhy } from "../app/services";
+import { clickOn } from "../test/testUtils";
+import { screen } from "@testing-library/react";
+import _ from "lodash";
```

Comment:
I don't think it really matters as I'm not sure this file gets bundled, but importing from lodash like this imports the whole (rather large) library. You can import individual functions via 
```suggestion
import zipWith from "lodash/zipWith";
```
and just use that function below. Even if it isn't necessary, it's a useful habit to get into!